src/InvestJob.js

```diff
@@ -0,0 +1,119 @@
+import crypto from 'crypto';
+import localforage from 'localforage';
+
+import { getLogger } from './logger';
+
+const logger = getLogger(__filename.split('/').slice(-1)[0]);
+
+const KEYS_ARRAY = 'sortedHashArray';
```

Comment:
Is `'sortedHashArray'` just the key name for the list of sorted keys?

Reply:
Yeah. That's a pretty ugly name, huh. Both the variable and the string. Maybe it would be better like
`const HASH_ARRAY_KEY = 'workspaceHashes'